contrib/docker/testnet.sh

```diff
@@ -74,6 +100,19 @@ set_repo()
   fi
 }
 
+set_bins()
+{
+  read_bool_input "Use binaries from repo?" KOVRI_USE_REPO_BINS ""
+
+  if [[ $KOVRI_USE_REPO_BINS = true ]];then
+    echo "Using binaries in ${KOVRI_REPO}/build"
+    mount_repo_bins="-v ${KOVRI_REPO}/build/kovri:/usr/bin/kovri \
+      -v ${KOVRI_REPO}/build/kovri-util:/usr/bin/kovri-util"
+
+    read_bool_input "Build repo binaries?" KOVRI_BUILD_REPO_BINS "Exec make"
```

Comment:
Re: our debugging in `#kovri-dev`, any objection to making this a `release-static` build?